src/engine/game/battle/encounter.lua

```diff
@@ -22,6 +22,11 @@
 local Encounter = Class()
 
 function Encounter:init()
+    -- Sets the encounter ID incase it was started without an encounter file
+    if not self.id then
+        self.id = "_local"
```

Comment:
what does "_local" mean? is this just a placeholder string? whats wrong with nil?

Reply:
> what does "_local" mean? is this just a placeholder string? whats wrong with nil?

It fixes a nil-concat error when calling Encounter:setFlag.
Admittedly, it would be better to make Encounter:setFlag set a flag on the initiating ChaserEnemy if it doesn't have an ID (and DOES have a ChaserEnemy)